src/server/pages.js

```diff
@@ -33,9 +33,22 @@ pages
   .add('editCollective', '/:slug/edit/:section?')
   .add('events', '/:collectiveSlug/events')
   .add('subscriptions', '/:collectiveSlug/subscriptions')
-  .add('orderCollectiveTier', '/:collectiveSlug/order/:TierId/:step?', createOrderPage)
+  .add(
+    'orderCollectiveOld',
+    '/:collectiveSlug/order/:TierId/:step(contributeAs|details|payment|submit)?',
+    createOrderPage,
+  )
   .add('orderEventTier', '/:collectiveSlug/events/:eventSlug/order/:TierId', 'createOrder') // New contribution flow not applied to events yet
-  .add('donate', '/:collectiveSlug/:verb(donate|pay|contribute)/:step?', createOrderPage)
+  .add(
+    'orderCollective',
+    '/:collectiveSlug/:verb(donate|pay|contribute)/:step(contributeAs|details|payment|submit)?',
```

Comment:
So there's no more `submit`step, it has been replaced by `success`

```suggestion
    '/:collectiveSlug/:verb(donate|pay|contribute)/:step(contributeAs|details|payment|success)?',
```